cmd/zetaclientd/start.go

```diff
@@ -225,37 +215,14 @@ func Start(_ *cobra.Command, _ []string) error {
 	go func() {
 		for {
 			time.Sleep(30 * time.Second)
-			ps := tssServer.GetKnownPeers()
-			metrics.NumConnectedPeers.Set(float64(len(ps)))
-			telemetryServer.SetConnectedPeers(ps)
-		}
-	}()
-	go func() {
-		host := tssServer.GetP2PHost()
-		pingRTT := make(map[peer.ID]int64)
-		pingRTTLock := sync.Mutex{}
-		for {
-			var wg sync.WaitGroup
-			for _, p := range whitelistedPeers {
-				wg.Add(1)
-				go func(p peer.ID) {
-					defer wg.Done()
-					ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
-					defer cancel()
-					result := <-ping.Ping(ctx, host, p)
-					pingRTTLock.Lock()
-					defer pingRTTLock.Unlock()
-					if result.Error != nil {
-						masterLogger.Error().Err(result.Error).Msg("ping error")
-						pingRTT[p] = -1 // RTT -1 indicate ping error
-						return
-					}
-					pingRTT[p] = result.RTT.Nanoseconds()
-				}(p)
-			}
-			wg.Wait()
-			telemetryServer.SetPingRTT(pingRTT)
-			time.Sleep(30 * time.Second)
+			knownPeersTss := tssServer.GetKnownPeers()
+			metrics.NumConnectedPeers.Set(float64(len(knownPeersTss)))
+			knownPeersAddrInfo := lo.Map(knownPeersTss, func(p tss.PeerInfo, _ int) peer.AddrInfo {
+				return peer.AddrInfo{
+					ID: peer.ID(p.ID),
+				}
+			})
```

Comment:
`tss.PeerInfo` doesn't quite fit into `peer.AddrInfo` so this is the best I can do.